extensions/oplogPopulator/OplogPopulator.js

```diff
@@ -200,7 +212,16 @@ class OplogPopulator {
                     '_id': 1,
                     'operationType': 1,
                     'documentKey._id': 1,
-                    'fullDocument.value': 1
+                    'fullDocument.value': 1,
+                    // transforming the BSON timestamp
+                    // into a usable date
```

Comment:
clusterTime should already be included, with a value like this, right?
```js
"clusterTime": Timestamp({ "t": 1666193824, "i": 1 }),
```

can't just read the timestamp (`clusterTime.t`) in oplogpopulator instead of converting to date string and parsing the date?

Reply:
I did this because i find it cleaner than the way we do in Arsenal which is not precise, i'd rather let mongo transform it into a usable timestamp.
```
 timestamp: new Date((itemObj.ts ? itemObj.ts.getHighBits() * 1000 : 0))
```

Reply:
And the value `clusterTime` is equal to:
```
Timestamp { _bsontype: 'Timestamp', low_: 1, high_: 1672913394 }
```